docs/extend/saved-objects.md

```diff
@@ -526,7 +526,7 @@ Forward compatibility schema can be implemented in two different ways.
     ```
 
     :::{note}
-    Even if highly recommended, implementing this schema is not strictly required. Type owners can manage unknown fields and inter-version compatibility themselves in their service layer instead.
+    Starting with {{kib}} 9.3.0, all new model versions must include a `forwardCompatibility` schema. This new requirement is designed to ensure that if a Serverless instance needs to be rolled back to an earlier version, the Saved Objects APIs will still deliver data in the original, pre-upgrade format, thereby maintaining API compatibility and safety during the rollback process.
```

Comment:
```suggestion
    :applies_to: stack: ga 9.3
    Starting with {{kib}} 9.3.0, all new model versions must include a `forwardCompatibility` schema. This new requirement is designed to ensure that if a Serverless instance needs to be rolled back to an earlier version, the Saved Objects APIs will still deliver data in the original, pre-upgrade format, thereby maintaining API compatibility and safety during the rollback process.
```
This adds a badge to the callout which is useful to locate the version-specific information